panopticum/management/commands/config/default.json

```diff
@@ -0,0 +1,305 @@
+{
+   "ComponentCategories":{
+      "Platform":{
+         "description":"",
+         "order":1
+      },
+      "Search":{
+         "description":"Search Services use this category",
+         "order":2
+      }
+   },
+   "ComponentSubCategories":[
+      {
+         "category":"SubCategory_A",
+         "description":"",
+         "name":"Platform"
+      },
+      {
+         "category":"SubCategory_B",
+         "description":"",
+         "name":"Platform"
+      },
+      {
+         "category":"SubCategory_B",
+         "description":"",
+         "name":"Search"
+      },
+      {
+         "category":"SubCategory_C",
+         "description":"",
+         "name":"Search"
+      }
+   ],
+   "ComponentDataPrivacies":{
+      "Framework":{
+         "order":1
+      },
+      "Infrastructure":{
+         "order":2
+      }
+   },
+   "ComponentRuntimes":[
+      "Library",
+      "Framework",
+      "Driver",
+      "OS Service",
+      "OS Process",
+      "Web Service",
+      "Database",
+      "MQ Service",
+      "Database",
+      "Component"
+   ],
+   "Loggers":[
+      "Logf"
+   ],
+   "Countries":[
+      "Russia",
+      "United Kingdom",
+      "Japan",
+      "Singapore",
+      "Germany",
+      "Estonia",
+      "France",
+      "United States",
+      "Korea",
+      "Switzerland",
+      "Italy",
+      "China",
+      "United States of America",
+      "Bulgaria",
+      "India",
+      "Australia",
+      "UNITED STATES",
+      "Belgium",
+      "Spain",
+      "Denmark",
+      "Romania",
+      "Malaysia",
+      "Belarus",
+      "Sweden",
+      "Hong Kong S.A.R.",
+      "Taiwan",
+      "Hong Kong SAR",
+      "Thailand",
+      "United Arab Emirates",
+      "Turkey",
+      "Brazil",
+      "Czech Republic",
+      "Israel",
+      "Netherlands",
+      "Ukraine",
+      "Poland",
+      "Indonesia",
+      "Hungary",
+      "South Africa",
+      "Sierra Leone",
+      "Kazakhstan"

```

Comment:
Sure will do

Reply:
I think we don't need to have countries as its not used at present. However even in case we keep it, we should use ISO3166 alpha3 codes.

At present, I considered to remove the list and take country alpha3 code directly from [iso3166 module](https://pypi.org/project/iso3166/).

Reply:
may be we should move countires creating to [init.json](https://github.com/perfguru87/panopticum/blob/master/panopticum/fixtures/init.json) fixture and create it after db migration at deployment stage?

Reply:
The reason why its not added to fixture is it overwrites the tables. Its better to use iso3166 module. But I really would like to drop this table as its not being used at the moment.